asciidoctorj-api/src/main/java/org/asciidoctor/ast/ContentNode.java

```diff
@@ -16,7 +16,7 @@ public interface ContentNode {
 
     ContentNode getParent();
 
-    String getContext();
```

Comment:
If I am not missing anything this would also remove access to the context of a PhraseNode.
But a PhraseNode even needs to get a context given when created by a Processor.

Reply:
Yes, I think I was wrong with my first assessment. It's fine to just add the function to the ContentNode.